Scripts/Install-DotNetTool.ps1

```diff
@@ -0,0 +1,48 @@
+param (
+    [Parameter(Mandatory = $true, HelpMessage = "The name of the tool to install.")]
+    [string]
+    $Name,
+
+    [Parameter(Mandatory = $true, HelpMessage = "The version of the tool to install.")]
+    [string]
+    $Version,
+
+    [Parameter(HelpMessage = "When present, the tool will be installed globally, locally otherwise.")]
+    [switch]
+    $Global
```

Comment:
Local is the default behavior, and the wrapper shouldn't change that.

Reply:
Why though? This is our script, primarily we'll use it, and in the context of GitHub Actions, where we only need global.

Reply:
I find it frustrating when a wrapper changes the behavior compared to what it's wrapping when there's a corresponding configuration option (even if it was the other way around, i.e., $Local switch), I think that's bad DX.
However, since we'll wrap it in its own action, so the context is surely GHA (not local development for example, where this script can still be useful), then I think it's OK there, because it needs to be --global.

Reply:
I see the point, and OK then.